File: Examples/CSharp/SMTP/SSLEnabledSMTPServer.cs
using System;
using System.Diagnostics;
using Aspose.Email.Mime;
using Aspose.Email.Clients.Smtp;
using Aspose.Email.Clients;

namespace Aspose.Email.Examples.CSharp.Email.IMAP
{
    class SSLEnabledSMTPServer
    {
        public static void Run()
        {        
            // The path to the File directory.
            string dataDir = RunExamples.GetDataDir_SMTP();
            string dstEmail = dataDir + "test-load.eml";

            // ExStart:SSLEnabledSMTPServer
            SmtpClient client = new SmtpClient("smtp.gmail.com");

            // Set username, Password, Port No, and SecurityOptions
            client.Username = "your.email@gmail.com";
            client.Password = "your.password";
            client.Port = 587;
            client.SecurityOptions = SecurityOptions.SSLExplicit;
            // ExEnd:SSLEnabledSMTPServer

            // Declare message as MailMessage instance
            MailMessage message = new MailMessage();

            // Use MailMessage properties like specify sender, recipient and message
            message.To = "newcustomeronnet@gmail.com";
            message.From = "newcustomeronnet@gmail.com";
            message.Subject = "Test Email";
            message.Body = "Hello World!";
            try
            {
                // Client will send this message
                client.Send(message);
                Console.WriteLine("Message sent");
            }

            catch (Exception ex)
            {
                Trace.WriteLine(ex.ToString());
            }

            Console.WriteLine(Environment.NewLine + "Email sent SSL successfully.");
        }
    }
}

File: Examples/VisualBasic/SMTP/SSLEnabledSMTPServer.vb
Imports System.IO
Imports Aspose.Email.Mail
Imports Aspose.Email.Outlook
Imports Aspose.Email.Pop3
Imports Aspose.Email
Imports Aspose.Email.Mime
Imports Aspose.Email.Imap
Imports System.Configuration
Imports System.Data
Imports Aspose.Email.Mail.Bounce
Imports Aspose.Email.Exchange
Imports Aspose.Email.Outlook.Pst

Namespace Aspose.Email.Examples.VisualBasic.Email.SMTP
    Public Class SSLEnabledSMTPServer
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_SMTP()
            Dim dstEmail As String = dataDir & Convert.ToString("test-load.eml")

            Dim client As New Aspose.Email.Mail.SmtpClient("smtp.gmail.com")

            ' Set username
            client.Username = "your.email@gmail.com"

            ' Set password
            client.Password = "your.password"

            ' Set the port to 587. This is the SSL port of SMTP server
            client.Port = 587

            ' Set the security mode to explicit
            client.SecurityOptions = SecurityOptions.Auto

            'Declare msg as MailMessage instance
            Dim msg As New MailMessage()

            'use MailMessage properties like specify sender, recipient and message
            msg.[To] = "newcustomeronnet@gmail.com"
            msg.From = "newcustomeronnet@gmail.com"
            msg.Subject = "Test Email"
            msg.Body = "Hello World!"

            Try
                'Client will send this message
                client.Send(msg)
                'Show only if message sent successfully
                System.Console.WriteLine("Message sent")

            Catch ex As System.Exception
                System.Diagnostics.Trace.WriteLine(ex.ToString())
            End Try

            Console.WriteLine(Environment.NewLine + "Email sent SSL successfully.")
        End Sub
    End Class
End Namespace